server/@types/express/index.d.ts

```diff
@@ -0,0 +1,27 @@
+export default {}
+
+declare module 'express-session' {
```

Comment:
copied from typescript template and modified for our request and session attributes